otterdog/eclipse-uprotocol.jsonnet

```diff
@@ -10,6 +10,9 @@ orgs.newOrg('automotive.uprotocol', 'eclipse-uprotocol') {
         "~DEFAULT_BRANCH"
       ],
       requires_linear_history: true,
+      required_pull_request+: {
+        required_approving_review_count: 1,
+      }
```

Comment:
If at all, I guess this should be added to the `pull_request_settings` rule.
But my understanding of the corresponding [GitHub documentation](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets#about-rule-layering) is, that this will effectively prevent any repositories in the org to actually use a setting of `0` (which we would like to be able to use) ...

Reply:
correct. Doing the fix